changelog/unreleased/issue-2944

```diff
@@ -0,0 +1,12 @@
+Enhancement: Backup options --files-from-verbatim and --files-from-raw
+
+The new backup flags ``--files-from-verbatim`` and ``--files-from-raw``
+read a list of files to save from a file. Unlike the existing ``--files-from``,
+these options cause the filenames to not be interpreted  as glob patterns.
+whitespace in filenames is preserved as-is and no pattern expansion is done.
+
+These flags are recommended over ``--files-from`` when generating the list of
+files to back up from a script.
```

Comment:
Syntax and grammar correction:

~~~
The new backup options `--files-from-verbatim` and `--files-from-raw`
reads a list of files to back up from a file. Unlike the existing `--files-from`,
these options does not interpret the listed filenames as glob patterns;
whitespace in filenames is preserved as-is and no pattern expansion is done.

These new flags are recommended over `--files-from` when generating the
list of files to back up from a script. Please see the documentation for specifics.
~~~

Reply:
Done, except that options really is plural (reads=>read, does=>do).